test/system/035-logs.bats

```diff
@@ -182,6 +182,7 @@ function _log_test_since() {
     before=$(date --iso-8601=seconds)
     run_podman run --log-driver=$driver -d --name test $IMAGE sh -c \
         "echo $s_before; trap 'echo $s_after; exit' SIGTERM; while :; do sleep 0.1; done"
+    wait_for_output "$s_before" test
```

Comment:
I suggest you drop the sleep 1 below now as this fix is much better.

Also I just looked in wait_for_output function maybe we should reduce the retry interval from 5 s to 1s? IT feel stupid to always wait at least 5s as it would slow the test just further down.

Reply:
We still need the `sleep 1` because we're checking with 1-second resolution, not NS. And, eek, 5 seconds??? Yes, I will fix that.

Reply:
Well I mean we can fix the date commands to return nanoseconds so we would not need the sleep.

Reply:
OK, thanks. I've pushed new changes with the 5->1 change, but keeping the 1-second resolution. I added a comment briefly explaining why. I don't feel strongly about it, though.

Reply:
I am fine to merge it like this.